app/components/VolunteerApproval/RequestingHours.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import { Button, ButtonToolbar, Input, OverlayTrigger, Popover} from 'react-bootstrap';
+
+var HourPopover =
+    <Popover id="hrconfirm" title="Hour Confirmation">
+        <div>
+            <h5>Start Time:</h5>
+            <h5>End Time:</h5>
+            <h5>Total: </h5>
+            <Button> Submit </Button>
+        </div>
+    </Popover>
```

Comment:
there are a few of these, these should all be functions since they will be consuming props, also avoid using vars

```javascript
const HourPopover = (props) => (
    <Popover id="hrconfirm" title="Hour Confirmation">
        <div>
             <h5>Start Time: {props.startTime}</h5>
             <h5>End Time: {props.endTime}</h5>
             <h5>Total:  {props.Total}</h5>
             <Button onClick={props.confirmHours(id)}> Submit </Button>
         </div>
     </Popover>
)
```